Makefile

```diff
@@ -35,7 +35,7 @@ endif
 ifeq ($(origin CXX),default)
   CXX ?= g++
 endif
-BENDER_VERSION = 0.28.1
+BENDER_VERSION = 0.28.2
```

Comment:
Please ensure the new Bender version works for both ModelSim and VCS simulations. I remember last time the Bender bugs caused the issue of Bender not being able to generate the correct "analysis.tcl" file for VCS format.